sos/report/__init__.py

```diff
@@ -680,6 +681,21 @@ def _get_network_namespaces(self):
                 out_ns.append(line.partition(' ')[0])
         return out_ns
 
+    def _get_devices_by_fstype(self):
+        _dev_fstypes = {}
+        _devs = sos_get_command_output("lsblk -nrpo FSTYPE,NAME")['output']
+        for line in (_devs.splitlines()):
+            helper = line.strip().split()
+            if len(helper) == 1:
+                helper.insert(0, 'unknown')
+            if "ext" in helper[0]:
+                helper[0] = 'ext2/3/4'
+            if helper[0] not in _dev_fstypes:
+                _dev_fstypes[helper[0]] = [helper[1]]
+            else:
+                _dev_fstypes[helper[0]].append(helper[1])
```

Comment:
My brain does not work on full speed at 10:30pm, but can't this be elegantly resolved by either

`_dev_fstypes[helper[0]] = `_dev_fstypes.get(helper[0], []).append(helper[1])`

or using `setdefault` method?